cwbi-auth-http-client/src/main/java/hec/army/usace/hec/cwbi/auth/http/client/trustmanagers/CwbiAuthTrustManager.java

```diff
@@ -58,7 +58,7 @@ private CwbiAuthTrustManager(TrustManagerFactory trustManagerFactory) {
     private static X509TrustManager buildTrustManager() {
         X509TrustManager retVal = null;
```

Comment:
Do you need this class anymore?

Reply:
There is some exception logging as well as a singleton instance that would be different from just using the default trust manager.